resources/js/components/VerificationForm.vue

```diff
@@ -59,12 +63,23 @@
                     code: '',
                     token: ''
                 }),
-                errorMessage: ''
+                errorMessage: '',
+                resend_timeout: 30
             }
         },
         props: [ 'token', 'mobile' ],
         created() {
             this.form.token = this.token;
+            setInterval(() => {
+                if (this.resend_timeout)
+                    this.resend_timeout--
+            }, 1000)
```

Comment:
Intervals should be cleared when finished. This will continue to run in the background